apps/frontend/src/libs/enums/app-route.enum.ts

```diff
@@ -1,7 +1,8 @@
 const AppRoute = {
+	APP: "/app",
 	ROOT: "/",
-	SIGN_IN: "/sign-in",
-	SIGN_UP: "/sign-up",
+	SIGN_IN: "sign-in",
+	SIGN_UP: "sign-up",
```

Comment:
hm, why this change is needed
```suggestion
	SIGN_IN: "/sign-in",
	SIGN_UP: "/sign-up",
```

Reply:
I see, I will take a look at this